Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM jinaai/jina:devel
+FROM jinaai/jina:latest
 
 RUN apt-get update && apt-get install -y jq curl
 
+RUN pip install docker
```

Comment:
Is this required because image `jinaai/jina:latest` doesn't ship with `jina[docker]`?

Reply:
yes it is